server/routes/applications.js

```diff
@@ -64,26 +65,42 @@ router.post('/manage-applications/add', (req, res, next) => {
       return res.redirect(302, '/manage-applications');
     }).catch(err => {
       viewData.this_errors = routeUtils.processException(err);
-      console.log(viewData.this_errors);
       res.render(`${routeViews}/add.njk`, viewData);
     });
 });
 
-router.get('/manage-applications/:appId/view', (req, res, next) => {
+router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
```

Comment:
This code (and a bunch of other code in this PR) is not part of `master`. Can we work off of a clean version of `master` please?